paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodInitialVisibilityTracker.kt

```diff
@@ -191,9 +195,9 @@ internal class PaymentMethodInitialVisibilityTracker(
 
         /**
          * Debounce delay after UI stabilizes before dispatching analytics event.
-         * 800ms allows bottom sheet animations and layout settling to complete
+         * 50ms allows bottom sheet animations and layout settling to complete
```

Comment:
Should we do this differently? Ideally we should kick off the event, and wait to send it for some delay, but cancel the previous one if it hasn't been sent due to the delay yet.